src/libostree/ostree-repo-commit.c

```diff
@@ -921,17 +913,26 @@ scan_loose_devino (OstreeRepo                     *self,
 
           dot = strrchr (name, '.');
           g_assert (dot);
-
+          
           if ((dot - name) != 62)
```

Comment:
Suggest a comment here as well.  This is the rest of the 64-digit hash, I assume.